Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -756,21 +770,19 @@ class DropDownMenu
     }
   }
 
-  private static Options_OnKeyDown(event: JQueryKeyEventObject, dropDownMenu: JQuery): void
+  private static Options_OnKeyDown (event: KeyboardEvent, dropDownMenu: HTMLElement): void
   {
-    ArgumentUtility.CheckNotNullAndTypeIsJQuery('dropDownMenu', dropDownMenu);
-
     if (DropDownMenu._currentPopup == null)
       return;
 
-    var itemInfos = DropDownMenu._menuInfos[dropDownMenu[0].id]!.ItemInfos!;
+    var itemInfos = DropDownMenu._menuInfos[dropDownMenu.id]!.ItemInfos!;
     if (itemInfos.length === 0)
       return;
 
     var oldIndex;
     var isSelectionUpdated = false;
-    var dropDownMenuItems = $(DropDownMenu._currentPopup).find('ul').children();
-    var currentItemIndex = $('.' + DropDownMenu._itemSelectedClassName, DropDownMenu._currentPopup).index();
+    var dropDownMenuItems = Array.from (DropDownMenu._currentPopup.querySelectorAll ('ul')).flatMap (el => Array.from (el.children));
+    var currentItemIndex = Array.from (document.querySelectorAll ('.' + DropDownMenu._itemSelectedClassName)).indexOf (DropDownMenu._currentPopup);
```

Comment:
Now that I look at it a second time, I think we have a problem here. `.index()` takes the first element in the jQuery object and determines its index regarding its parent. So what we want to do here is find any selected item and determine its index or -1 if no item is selected.

Something like:
```js
const selectedElement = DropDownMenu._currentPopup.querySelector('.' + DropDownMenu._itemSelectedClassName);
const selectedElementIndex = selectedElement ? Array.from(selectedElement .parentNode.children).indexOf(selectedElement) : -1;

```

Reply:
Makes sense